tests/test_recipes.py

```diff
@@ -16,19 +16,38 @@ def test_recipe_registration(basicApp):
 def test_recipe_build(basicApp, tmpdir):
     basicApp.plugins.get("RecipePlugin")
     test_recipe = basicApp.recipes.get("test_recipe")
+    output_folder = str(tmpdir.mkdir("output"))
+    output_data = test_recipe.build(output_dir=output_folder, no_input=True, extra_context=None)
```

Comment:
Try not to update existing test cases.

Or in this case test also with:
`output_data = test_recipe.build(output_folder)`
to prove the old API is still valid

Reply:
Corrected